### DISCUSSION TITLE
Cloning/Duplication of a form section

### DISCUSSION THREAD

**Discussion by bynicolas** [statamic/cms#6205] [Q&A] [answered]
Hi,

I was wondering if cloning/adding a set of inputs within a form, like a section or a row, is an available feature built in?

Think of something like a resume form that would have `0 to n` experience sections, but would only show if the applicant has something to share. A button would add a new row with specific fields for each past experience the applicant has.

I was looking into the replicator field type since it's doing pretty much that, but after reading the docs, it looks like it's more of a control panel feature rather than one for the front end.

Am I wrong or am I missing something and the replicator can be used dynamically in the front end as well?

Thanks in advance for the help.

**Comment by jasonvarga** [comment, marked as answer]
You can use a grid or replicator, but you'd have to replicate (hah) it yourself on the frontend using some javascript.

**Reply by bynicolas** [reply]
Thanks for the clarification, it gets me on the right track!
